.github/workflows/unit-test.yml

```
@@ -2,20 +2,138 @@ name: Unit Test
 jobs:
   unit-test:
     permissions: write-all
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
+    - name: Install Trivy
+      run: |
+        curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.48.1
     - name: Setup GitHub Action
       uses: actions/setup-node@v1
```

actionlint (1.7.4)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)